### ISSUE TITLE
Only UMass students should be allowed to post reviews

### ISSUE THREAD

**Issue by jrbartola** [ratemyclass/RateMyClass#2]
This issue is a bit more advanced. Ideally, we would not want anybody on the internet to be able to post a review for a UMass class if they don't attend the university. Perhaps we could request that they verify an @umass.edu email address before being given the opportunity to compose a new review.

**Comment by jeffrey-carr** [comment]
Anyone in the 5-college system can take UMass classes. Maybe check for all those emails?

**Comment by jrbartola** [comment]
Right- good point. I think we should have some configuration file that contains which email addresses are allowed for whatever university is using the system
